ledger/ledger-api-client/src/main/scala/com/digitalasset/util/Ctx.scala

```diff
@@ -25,10 +25,6 @@ final case class Ctx[+Context, +Value](
 
 object Ctx {
 
-  def fromPair[Context, Value](pair: (Context, Value)) = Ctx(pair._1, pair._2)
```

Comment:
Unused

Reply:
This change is unrelated to the metrics change. Could you please split it off to a separate PR?